packages/components/src/Emphasis/Emphasis.component.js

```diff
@@ -0,0 +1,31 @@
+import React, { PropTypes } from 'react';
+import theme from './Emphasis.scss';
+
+function emphasise(text, value) {
+	if (!text) {
+		return '';
+	}
+	if (!value) {
+		return text;
+	}
+
+	const parts = text.split(new RegExp(`(${value})`, 'gi')).filter(Boolean);
```

Comment:
That's a lot of operation in one line. 
I don't really get the purpose of the last filter.